### DISCUSSION TITLE
Statmic and Laravel Breeze using both files and database to store users

### DISCUSSION THREAD

**Discussion by joshuapphillips** [statamic/cms#7274] [Q&A] [answered]
Is it possible to use continue to use the file based users for the statmic control panel, but install Larvel Breeze and use a database for customers?

I'd like to keep the two types of user completely separate.

How do I implement this?

**Comment by duncanmcclean** [comment, marked as answer]
Yes, that's possible. You can use an [independent auth guard for Statamic users](https://statamic.dev/tips/using-an-independent-authentication-guard).
